### ISSUE TITLE
Find broken links

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutisticaCitizenScience#425]
## Summary

If you find a broken link anywhere in the Autistica GitHub let us know by writing where it is or copying the url in a comment below 👇 and we will fix it ASAP. Thank you!! 

### Want to fix the links yourself? 

Go to issue [#423](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/423)

**Comment by GeorgiaHCA** [comment]
The link to "community-guidelines" in the 'Get Involved' section is broken

**Comment by SuziQpid** [comment]
> ## Summary
> If you find a broken link anywhere in the Autistica GitHub let us know by writing where it is or copying the url in a comment below 👇 and we will fix it ASAP. Thank you!!
> 
> ### Want to fix the links yourself?
> Go to issue [#423](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/423)

Finally got in. Clueless how to go back and copy url. Don’t want to jink it. Following repository link you sent me via email it brought up log in, did usual of challenging my user name. Went back to double check on my iPad saved section in settings. I had stupidly put a gap in my user name, edited that. Went back and l had a 404 error page in place of login. Pasted repository link in main search part of git hub page brought up 404 again. Reported problem to GitHub myself. Said I’d try one more time. This time went back to google search to get a general link address to sign in. Tickety boo, got a sign in page, then went to a verification code from my inbox, pasted repository link and here l am. Not sure l’ve put this in the right place. It’s an accessibility problem and bug. The more l use the platform the easier it will get. However l do think there’s a bug in the app when trying to log in on an iPad. I did mention in my report to GitHub that l was on an iPad. Let’s see what they say after l abruptly declined being helped by a virtual assistant. I love A.I. until tech fails then l need a person !!!!
